elyra/pipeline/airflow/processor_airflow.py

```diff
@@ -400,13 +400,17 @@ def create_pipeline_file(self, pipeline, pipeline_export_format, pipeline_export
             if pipeline_description is None:
                 pipeline_description = f"Created with Elyra {__version__} pipeline editor using `{pipeline.source}`."
 
+            schedule_interval = pipeline.schedule_interval
```

Comment:
Would it be better to have this in pipeline definition or pipeline value object directly?